cmd/nephe-controller/main.go

```diff
@@ -56,6 +57,15 @@ func main() {
 	var metricsAddr string
 	var enableLeaderElection bool
 	var enableDebugLog bool
+	opts := newOptions()
+
+	flag.StringVar(&opts.configFile, "config", opts.configFile, "The path to the configuration file.")
+	err := opts.complete()
+	if err != nil {
+		setupLog.Error(err, "Invalid controller config map data")
```

Comment:
All error message should start with smallcase. 
```suggestion
		setupLog.Error(err, "invalid config map data")
```

Reply:
Solved